### PR TITLE
Add deprecated functions from interactivity core blocks

### PR DESCRIPTION
**Pull request by cbravobernal**
Add Interactivity functions to deprecated file.

Trac ticket: https://core.trac.wordpress.org/ticket/60380

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/deprecated.php [hunk 1]
```diff
@@ -6239,3 +6239,60 @@ function the_block_template_skip_link() {
...
+/**
+ * Ensure that the view script has the `wp-interactivity` dependency.
+ *
+ * @since 6.4.0
+ * @deprecated 6.5.0
+ *
+ * @global WP_Scripts $wp_scripts
+ */
+function block_core_query_ensure_interactivity_dependency() {
+	_deprecated_function( __FUNCTION__, '6.5.0', 'Used wp_register_script_module instead' );
```
Comment (swissspidy) [inline src/wp-includes/deprecated.php]:
```suggestion
	_deprecated_function( __FUNCTION__, '6.5.0', 'wp_register_script_module' );
```

File: src/wp-includes/deprecated.php [hunk 2]
```diff
@@ -6239,3 +6239,60 @@ function the_block_template_skip_link() {
...
+/**
+ * Ensure that the view script has the `wp-interactivity` dependency.
+ *
+ * @since 6.4.0
+ * @deprecated 6.5.0
+ *
+ * @global WP_Scripts $wp_scripts
+ */
+function block_core_file_ensure_interactivity_dependency() {
+	_deprecated_function( __FUNCTION__, '6.5.0', 'Used wp_register_script_module instead' );
```
Comment (swissspidy) [inline src/wp-includes/deprecated.php]:
```suggestion
	_deprecated_function( __FUNCTION__, '6.5.0', 'wp_register_script_module' );

File: src/wp-includes/deprecated.php [hunk 3]
```diff
@@ -6239,3 +6239,60 @@ function the_block_template_skip_link() {
...
+/**
+ * Ensures that the view script has the `wp-interactivity` dependency.
+ *
+ * @since 6.4.0
+ * @deprecated 6.5.0
+ *
+ * @global WP_Scripts $wp_scripts
+ */
+function block_core_image_ensure_interactivity_dependency() {
+	_deprecated_function( __FUNCTION__, '6.5.0', 'Used wp_register_script_module instead' );
```
Comment (swissspidy) [inline src/wp-includes/deprecated.php]:
```suggestion
	_deprecated_function( __FUNCTION__, '6.5.0', 'wp_register_script_module' );
```

File: src/wp-includes/deprecated.php (hunk 1 above)
Comment (cbravobernal) [reply src/wp-includes/deprecated.php]: Done in https://github.com/WordPress/wordpress-develop/pull/5973/commits/b6af53403e501bb2dc8d7083a16d657bc1c53b55

File: src/wp-includes/deprecated.php (hunk 2 above)
Comment (cbravobernal) [reply src/wp-includes/deprecated.php]: Done in https://github.com/WordPress/wordpress-develop/pull/5973/commits/b6af53403e501bb2dc8d7083a16d657bc1c53b55

File: src/wp-includes/deprecated.php (hunk 3 above)
Comment (cbravobernal) [reply src/wp-includes/deprecated.php]: Done in https://github.com/WordPress/wordpress-develop/pull/5973/commits/b6af53403e501bb2dc8d7083a16d657bc1c53b55

**Comment by youknowriad** [comment]
commit https://core.trac.wordpress.org/changeset/57511

**Pull request closed without merging**